react.mdx

```diff
@@ -174,4 +233,39 @@ React requires Javascript to show the content on the page. The problem is that t
 
 ## Questions
 
-<img src={Questions} />
+<img style={{ height: '50%' }} src={Questions} />
+
+---
+
+<LiveCode
+  title="Hooks version"
+  code={`
+  // ------ Functional ------
+  const Title = (props) => (
+    <h3 style={{ color: 'palevioletred' }}>
+      {props.title}
+    </h3>
+  );
+  // ---- Functional with Hooks ----
+  const Likes = () {
+    const [likes, setLikes] = useState(0)
+    const increaseSelfEsteem = () => setLikes(likes + 1)
+    return (
+      <>
+        <h3>{likes} 🖤</h3>
+        <button onClick={increaseSelfEsteem}>
+          Like me 🥺
+        </button>
+      </>  
+    )
+  }
+  render(
+    <center>
+      <Title title="Hello World!" />
+      <Likes />
+    </center>
+  )
+  `} 
+  providerProps={{ noInline: true }}
```

Comment:
If you throw a `import {useState} from 'react'` at the top of the file, then I got this to work locally:
```suggestion
  providerProps={{ scope: {useState}, noInline: true }}
```